src/components/checkbox/checkbox.js

```diff
@@ -16,7 +16,7 @@ angular
  * @description
  * The checkbox directive is used like the normal [angular checkbox](https://docs.angularjs.org/api/ng/input/input%5Bcheckbox%5D).
  *
- * As per the [material design spec](http://www.google.com/design/spec/style/color.html#color-color-schemes)
+ * As per the [material design spec](https://material.io/archive/guidelines/style/color.html#color-color-palette)
```

Comment:
Let's capitalize `Material Design` here.

Reply:
I fixed this and changed two other comments with the same structure for consistency.